applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/spy_condition.cpp

```diff
@@ -0,0 +1,78 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#include "spy_condition.h"
+
+namespace Kratos::Testing {
+
+void SpyCondition::FinalizeSolutionStep(const ProcessInfo& rCurrentProcessInfo)
+{
+    mSolutionStepFinalized = true;
+}
+
+void SpyCondition::InitializeSolutionStep(const ProcessInfo& rCurrentProcessInfo)
+{
+    mSolutionStepInitialized = true;
+}
```

Comment:
I would swap these two function definitions, and remove the unused parameter names:
```suggestion
void SpyCondition::InitializeSolutionStep(const ProcessInfo&)
{
    mSolutionStepInitialized = true;
}

void SpyCondition::FinalizeSolutionStep(const ProcessInfo&)
{
    mSolutionStepFinalized = true;
}
```
The latter suggestion also applies to the other member functions.

Reply:
Agreed, changed